commcare_connect/templates/account/login.html

```diff
@@ -1,63 +1,82 @@
     {% endif %}
-  </p>
-
-  <div class="socialaccount_ballot">
-
-    <ul class="socialaccount_providers">
-      {% include "socialaccount/snippets/provider_list.html" with process="login" %}
-    </ul>
-
-    <div class="login-or text-center mb-2">{% translate "or" %}</div>
+    <h6 class="title text-brand-deep-purple">Login</h6>
+    <span class="text-sm text-gray-600">Please enter your details</span>
 
-  </div>
-
-  {% include "socialaccount/snippets/login_extra.html" %}
-
-{% else %}
-  {% if ACCOUNT_ALLOW_REGISTRATION %}
-    <p>
-      {% blocktranslate trimmed %}
-        If you have not created an account yet, then please
-        <a href="{{ signup_url }}">sign up</a> first.
-      {% endblocktranslate %}
-    </p>
-  {% endif %}
-{% endif %}
-
-<form class="login" method="POST" action="{% url 'account_login' %}">
-  {% csrf_token %}
-  {{ form|crispy }}
-  {% if redirect_field_value %}
-    <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
-  {% endif %}
-
-  <button class="primaryAction btn btn-primary w-100" type="submit">{% translate "Sign In" %}</button>
+    {% if form.non_field_errors %}
+      {% for error in form.non_field_errors %}
+        <p class="text-sm text-red-600">* {{ error }}</p>
+      {% endfor %}
+    {% endif %}
 
-  <div class="text-center mt-2">
-    <a class="button secondaryAction " href="{% url 'account_reset_password' %}"><small>{% translate "Forgot Password?" %}</small></a>
-  </div>
+    <div class="input-group {% if form.login.errors %}error{%endif%}">
+        <div class="simple-input">
+            <input type="email"
+                   placeholder="Enter Email ID"
+                   required
+                   id="{{ form.login.auto_id }}"
+                   {% if form.login.value != None %}value="{{ form.login.value|stringformat:'s' }}"{% endif %}
+                   name="{{ form.login.html_name }}">
+            <i class="fa-light fa-envelope"></i>
+            <span class="error-icon fa-solid fa-circle-exclamation"></span>
+        </div>
+        {% for error in form.login.errors %}
+          <p class="error-msg">* {{ error }}</p>
+        {% endfor %}
+    </div>
+
+    <div class="input-group {% if form.password.errors %}error{%endif%}">
+        <div class="simple-input">
+            <input type="password"
+                   placeholder="Password"
+                   required
+                   id="{{ form.password.auto_id }}"
+                   {% if form.password.value != None %}value="{{ form.password.value|stringformat:'s' }}"{% endif %}
+                   name="{{ form.password.html_name }}">
+            <i class="fa-light fa-lock"></i>
+            <span class="error-icon fa-solid fa-circle-exclamation"></span>
+        </div>
+        {% for error in form.password.errors %}
+          <p class="error-msg">* {{ error }}</p>
+        {% endfor %}
+    </div>
```

Comment:
why render this form manually instead of using crispy?

Reply:
The fields have custom css and no labels. I was not sure if that would be possible if we render this through crispy.